docs/release_notes/6.0.4-3.md

```diff
@@ -0,0 +1,26 @@
+# Percona Server for MongoDB 6.0.4-3 (2023-01-26)
+
+| Release date | January 26, 2023|
+|------------- | ---------------|
+| **Installation** | [Installing Percona Backup for MongoDB](../install/index.md) |
+
+
+Percona Server for MongoDB 6.0.4-3 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 6.0.4 Community Edition.
+
+It is rebased on [MongoDB 6.0.4 Community edition](https://www.mongodb.com/docs/manual/release-notes/6.0/#6.0.4-rc0---jan-12--2023) and supports the upstream protocols and drivers.
+
+## Release Highlights
+
+* Percona Server for MongoDB is now available on Red Hat Enterprise Linux 9 and compatible derivatives
+* Percona Server for MongoDB is now available on ARM64 architectures
```

Comment:
Only docker image for ARM will be available and AFAIK it will be technical preview feature, need to check with @janwieremjewicz

Reply:
Updated as RC

Reply:
Thanks!